docs/uaa.md

```diff
@@ -0,0 +1,23 @@
+# UAA
+
+To target and authenticate with the UAA:
+
+```plain
+cf_values=/tmp/cf-values.yml
+
+uaa target  "https://uaa.$(bosh int "$cf_values" --path /system_domain)" --skip-ssl-validation
```

Comment:
I'd personally prefer a `yq` command over the bosh one.

`yq -r .system_domain "$cf_values"`